extensions/default/src/DicomWebDataSource/utils/clientManager.ts

```diff
@@ -0,0 +1,210 @@
+import { api } from 'dicomweb-client';
+import StaticWadoClient from './StaticWadoClient';
+import dcm4cheeReject from '../dcm4cheeReject';
+import { errorHandler, utils } from '@ohif/core';
+
+export default class clientManager {
+  clients;
+  userAuthenticationService;
+
+  // check google server params
+  private checkGoogleServerParams(params) {
```

Comment:
I don't think we can have google or gcp inside a generic dicomweb data source. You need to create a new datasource under a new extension specifically made for gcp or made this more generic.